face_detection_masking/main.py

```diff
@@ -186,19 +185,25 @@ def process_img(
         bottom_right_y = int(bottom_right_y) + int(make_square)
 
     # Make the background color grey
-
     # naive fix for linting error: line getting to long ->
     crop_img = masked_img[top_left_y:bottom_right_y, top_left_x:bottom_right_x].copy()
     crop_img[np.where((crop_img == [0, 0, 0]).all(axis=2))] = [140, 141, 137]
 
     # Convert to Grayscale
+
```

Comment:
?

Reply:
Oh crap! The comment is left from the time when we were having conflicts between flake8 and black linting standards. I will remove the misleading comment.